packages/sui-mono/bin/sui-mono-release.js

```diff
@@ -69,20 +69,26 @@ const releaseEachPkg = ({pkg, code} = {}) => {
     const pkgInfo = require(path.join(cwd, 'package.json'))
     const scripts = pkgInfo.scripts || {}
 
-    let commands = [
+    let releaseCommands = [
       ['npm', ['--no-git-tag-version', 'version', `${RELEASE_CODES[code]}`]],
       ['git', ['add', cwd]],
-      ['git', ['commit -m "release(' + packageScope + '): v$(node -p -e "require(\'./package.json\')".version)"']],
+      ['git', ['commit -m "release(' + packageScope + '): v$(node -p -e "require(\'./package.json\')".version)"']]
+    ]
+    let docCommands = [
       [suiMonoBinPath, ['changelog', cwd]],
       ['git', ['add', cwd]],
-      ['git', ['commit --amend --no-verify --no-edit']],
+      ['git', ['commit --amend --no-verify --no-edit']]
+    ]
+    let publishCommands = [
       ['git', ['tag -a ' + tagPrefix + '$(node -p -e "require(\'./package.json\')".version) -m \"v$(node -p -e "require(\'./package.json\')".version)\"']], // eslint-disable-line no-useless-escape
+      scripts['build'] && ['npm', ['run', 'build']],
       !pkgInfo.private && ['npm', ['publish', `--access=${publishAccess}`]],
       ['git', ['push', '--tags', 'origin', 'HEAD']]
     ].filter(Boolean)
-    scripts['build'] && commands.unshift(['npm', ['run', 'build']])
 
-    serialSpawn(commands, {cwd})
+    serialSpawn(releaseCommands, {cwd})
+      .then(() => serialSpawn(docCommands))
```

Comment:
Maybe I don't understand the problem.

This is because we want to generate the CHANGELOG for all component on the root package?

Reply:
We want to generate the log for one package, but with the monorepo as cwd.
This way, when isMonoPackage is executed, it would return false, not true.